Tasks/NuGetCommand/nugetcommandmain.ts

```diff
@@ -52,4 +55,81 @@ async function main(): Promise<void> {
     }
 }
 
+
+function _logNugetStartupVariables() {
+    try {
+        let externalendpoint = null;
+        let epId = tl.getInput('externalendpoint');
+        if (epId) {
+            externalendpoint = {
+                feedName: tl.getEndpointUrl(epId, true).replace(/\W/g, ''),
+                feedUri: tl.getEndpointUrl(epId, true)
+            }
+        }
+        let externalendpoints = tl.getDelimitedInput('externalendpoints', ',');
+        if (externalendpoints) {
+            externalendpoints = externalendpoints.reduce((ary, id) => {
+                let te = {
+                    feedName: tl.getEndpointUrl(id, true).replace(/\W/g, ''),
+                    feedUri: tl.getEndpointUrl(id, true)
+                }
+                ary.push(te);
+                return ary;
+            }, []);
+        }
+        let nugetTelemStr = JSON.stringify(
+            {
+                'command': tl.getInput('command'),
+                'arguments': tl.getInput('arguments'),
+                'NUGET_EXE_TOOL_PATH_ENV_VAR': tl.getVariable(nuGetGetter.NUGET_EXE_TOOL_PATH_ENV_VAR),
+                'NUGET_EXE_CUSTOM_LOCATION': tl.getVariable(NUGET_EXE_CUSTOM_LOCATION),
+                'searchPatternPack': tl.getPathInput('searchPatternPack'),
+                'configurationToPack': tl.getInput('configurationToPack'),
+                'versioningScheme': tl.getInput('versioningScheme'),
+                'includeReferencedProjects': tl.getBoolInput('includeReferencedProjects'),
+                'versionEnvVar': tl.getInput('versionEnvVar') ?
+                    tl.getInput(tl.getInput('versionEnvVar')) : null,
```

Comment:
This looks weird, why is this needed?

Reply:
Fixed.  Should be getVariable(getInput).  This is necessary when a user supplies an input variable containing the name of the environment variable with the actual version.